/var/spool/slurmd/job82635/slurm_script: line 11: cd: SSL_VAN: No such file or directory

----------------------------------------------------------------------------
  cuda:
----------------------------------------------------------------------------
     Versions:
        cuda/10.2
        cuda/11.0
        cuda/11.2
        cuda/11.3
        cuda/11.4
        cuda/11.5
        cuda/11.6
        cuda/11.7
        cuda/11.8
        cuda/12.0

----------------------------------------------------------------------------
  For detailed information about a specific "cuda" package (including how to load the modules) use the module's full name.
  Note that names that have a trailing (E) are extensions provided by other modules.
  For example:

     $ module spider cuda/12.0
----------------------------------------------------------------------------

 


CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
srun: error: a100-10: tasks 0-4,7-12,14-17,20-24,26-30,32-52,54-55,57-58,60-63: Exited with exit code 1
srun: error: a100-10: tasks 5-6,13,18-19,25,31,53,56,59: Exited with exit code 1
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
Traceback (most recent call last):
  File "/home/karimimonsefi.1/SSL_VAN/BTCV/main.py", line 11, in <module>
    from BTCV.utils.data_utils import get_loader
ModuleNotFoundError: No module named 'BTCV'
srun: error: a100-12: tasks 64-80,82-97,100-127: Exited with exit code 1
srun: error: a100-12: tasks 81,98-99: Exited with exit code 1
